daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -1018,11 +966,9 @@ private[lf] object Speedy {
     def fromScenarioSExpr(
         compiledPackages: CompiledPackages,
         scenario: SExpr,
-        disclosedContracts: ImmArray[speedy.DisclosedContract] = ImmArray.Empty,
```

Comment:
As contract disclosures are on ledger only, removed disclosure arguments from this call and elsewhere